src/store/actions/user.js

```diff
@@ -80,3 +78,18 @@ export const updateUser = (data, password) => {
     return dispatch(frApiRequest(actionTypes.users.update, request))
   }
 }
+
+export const updateAvatar = (user, datamime, data) => {
+  return (dispatch, getState) => {
+    const formData = new FormData();
+    formData.append('image', data)
+    const request = {
+      url: `/users/${user}/image`,
+      method: 'post',
+      contenttype: datamime,
```

Comment:
content type cannot be set like this, and since this is formData it doesn't matter anyway. There's no need to really care about the mime type of the input image